tests/test_isolanguage.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2016-2020 CERN.
+#
+# Marshmallow-Utils is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Test the marshmallow ISO Language."""
+
+import pytest
+from marshmallow import Schema, ValidationError, fields
+
+from marshmallow_utils.fields import isolanguage
+
+
+def test_iso639_3():
+
+    class MySchema(Schema):
+        f = isolanguage.ISOLangString()
+
+    assert MySchema().load({'f': "tes"}) == {'f': "tes"}
+
+    pytest.raises(ValidationError, MySchema().load, {'f': "t"})
```

Comment:
moderate: it might be a good idea to refactor this using parametrized input. I think you *might* need to split in two tests since one is a `raises` and the other asserts.

See https://docs.pytest.org/en/stable/parametrize.html

Reply:
Addressed :)